processor/worker.go

```diff
@@ -72,7 +72,11 @@ func (w *worker) start() {
 		defer close(w.channel.transform)
 		defer w.logger.Debugf("preprocessing routine stopped for worker: %s", w.partition)
 		for jobs := range w.channel.preprocess {
-			w.channel.transform <- w.handle.processJobsForDest(w.partition, jobs)
+			val, err := w.handle.processJobsForDest(w.partition, jobs)
```

Comment:
do we also want to use `processJobsForDestv2` here?

Reply:
Fixed